src/transformers/Streamable.js

```diff
@@ -0,0 +1,9 @@
+import fetch from 'node-fetch';
+
+export const shouldTransform = url =>
+  /^https?:\/\/streamable\.com\/?[|s|o|\D].+/.test(url);
```

Comment:
Make this more readable by using the `url` dependency, just like we do in all our other transformers:
- Check if the `host` is 'streamable.com' or 'www.streamable.com'
- Check the length of the `/` splitted `pathname`
  - If `1`
    - Return `true`
  - If `2` or `3`
    - Return if the first part is one of `'/o/'` or `'/s/'` (copy over the `includesSomeOfArray` from the Spotify transformer, we'll extract that into a helper function in a separate PR)
  - Else
    - Return `false`